components/Header/index.vue

```diff
@@ -11,9 +10,27 @@
         </BaseButton>
       </div>
       <div class="header__action">
-        <BaseButton variant="tertiary" type="button" label="Sayembara Desa Digital" @click="$router.push('/sayembara')" />
+        <BaseButton
+          :class="{
+            'header__button': true,
+            'header__button--actived': activeRoute === 'sayembara'
+          }"
+          variant="tertiary"
+          type="button"
+          label="Sayembara Desa Digital"
+          @click="$router.push('/sayembara')"
+        />
         <span class="text-gray-300">|</span>
-        <BaseButton variant="tertiary" type="button" label="Tentang Desa Digital" @click="$router.push('/about')" />
+        <BaseButton
+          :class="{
+            'header__button': true,
+            'header__button--actived': activeRoute === 'about'
+          }"
+          variant="tertiary"
+          type="button"
+          label="Tentang Desa Digital"
+          @click="$router.push('/about')"
+        />
```

Comment:
super minor:

hello @yoslie , In my opinion It's easier to use `<nuxt-link>` as a button because it can detect active links automatically, we just need to add styling for it

https://nuxtjs.org/examples/routing/active-link-classes/

ps: we don't have to change this, just an alternative approach 👍🏼

Reply:
sure I'll use this in advance, thankyou 🙏